### PR TITLE
New readme.

### PR DESCRIPTION
**Pull request by coderanger**
Ping @mahmoudimus @bninja @mjallday

### CODE REVIEW

**Comment by mahmoudimus** [comment]
how to pack?

**Comment by mahmoudimus** [comment]
how to use cloudformation w/ brix w/o using strataosphere?

File: README.md [hunk 1]
```diff
@@ -1,34 +1,84 @@
...
+The update subcommand with create or update a stack using a given template. The
+`--no-sync` argument will suppress the initial sync to S3. Be warned this can
+result in errors if you reference an un-synced template from your stack. The
+`--param` argument can be used to pass parameters to the stack. When updating an
+existing stack, all existing parameters will be copied over.
 
 Adding A Template
 -----------------
 
 To add a new template you need to:
```
Comment (bninja) [inline README.md:80]: doesnt render as list for me

**Comment by coderanger** [comment]
Docs added for packer, will hold off on the non-stratosphere docs until someone actually tries it :-)

**Pull request merged**